tests/tensor/rewriting/test_subtensor.py

```diff
@@ -2402,3 +2402,24 @@ def test_local_blockwise_advanced_inc_subtensor(set_instead_of_inc):
     else:
         expected_out[:, :, core_idxs] += test_y
     np.testing.assert_allclose(fn(test_x, test_y), expected_out)
+
+
+@pytest.mark.parametrize("fstop, lstop, lstep", [(None, 9, 1), (-1, -1, -1)])
```

Comment:
Parametrize with the manipulation and the expectation, or don't parametrize and just run one after the other. This way the logic is broken across different parts and harder to read